x-pack/solutions/security/test/cloud_security_posture_api/routes/graph.ts

```diff
@@ -694,6 +697,247 @@ export default function (providerContext: FtrProviderContext) {
+          const customNamespaceId = 'test';
+          // Create a test space
+          await spacesService.create({
+            id: customNamespaceId,
+            name: `${customNamespaceId} namespace`,
+            solution: 'security',
+            disabledFeatures: [],
+          });
+
+          await kibanaServer.uiSettings.update(
+            { 'securitySolution:enableAssetInventory': true },
+            { space: customNamespaceId }
+          );
+
+          // Create the data view in the test space
+          const dataView = dataViewRouteHelpersFactory(supertest, customNamespaceId);
+          await dataView.create('security-solution');
+
+          // Enable asset inventory in the test space
+          await supertest
+            .post(`/s/${customNamespaceId}/api/asset_inventory/enable`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({})
+            .expect(200);
+
+          const data = await supertest
+            .get(`/s/${customNamespaceId}/api/fleet/epm/packages/cloud_asset_inventory`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({})
+            .expect(200);
+
+          await supertest
+            .post(`/s/${customNamespaceId}/api/fleet/epm/packages/_bulk`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({
+              packages: [
+                {
+                  name: 'cloud_asset_inventory',
+                  version: data.body.item.version,
+                },
+              ],
+            })
+            .expect(200);
+
+          await retry.tryForTime(enrichPolicyCreationTimeout, async () => {
+            const response = await postGraph(
+              supertest,
+              {
+                query: {
+                  indexPatterns: ['logs-*'],
+                  originEventIds: [],
+                  start: '2024-09-01T00:00:00Z',
+                  end: '2024-09-02T00:00:00Z',
+                  esQuery: {
+                    bool: {
+                      filter: [
+                        {
+                          match_phrase: {
+                            'actor.entity.id': 'admin@example.com',
+                          },
+                        },
+                      ],
+                    },
+                  },
+                },
+              },
+              undefined,
+              customNamespaceId
+            ).expect(result(200, logger));
+
+            const actorNode = response.body.nodes.find(
+              (node: NodeDataModel) => node.id === 'admin@example.com'
+            ) as EntityNodeDataModel;
+
+            // Verify entity enrichment
+            expect(actorNode).not.to.be(undefined);
+            expect(actorNode.label).to.equal('AdminExample');
+            expect(actorNode.icon).to.equal('user');
+            expect(actorNode.shape).to.equal('ellipse');
+            expect(actorNode.tag).to.equal('Identity');
+
+            // Clean up the test space
+            await spacesService.delete(customNamespaceId);
```

Comment:
cleanup should be part of clean section - not part of the test.

this way, on test failure, you don't contaminate the testing environment